src/main/groovy/net/minecraftforge/gradleutils/changelog/ChangelogUtils.groovy

```diff
@@ -153,7 +153,8 @@ class ChangelogUtils {
         return remoteBranches.stream()
                 .filter(branch -> branch.getObjectId().getName() != headCommit.toObjectId().getName())
                 .map(branch -> getMergeBase(git, branch))
-                .filter(revCommit -> revCommit.toObjectId().getName() != headCommit.toObjectId().getName())
+                .filter(revCommit -> (revCommit != null) &&
```

Comment:
```suggestion
                .filter(revCommit -> (revCommit !== null) &&
```
In Groovy, `!=` is roughly equivalent to `!obj.equals()`. To check for null, do `!==` instead - it's faster and prevents people from overriding the equals method to screw with these kinds of checks.